docs/pages/includes/desktop-access/desktop-config.yaml

```
@@ -31,6 +31,12 @@ windows_desktop_service:
     insecure_skip_verify: false
     # DER encoded LDAP CA certificate.
     der_ca_file: /path/to/cert
+  # (optional) hosts is a list of hostnames to register as WindowsDesktop
+  # objects in Teleport. These are usually non-AD hosts.
```

```suggestion
  # objects in Teleport.
```

We don't support non-AD hosts.